Nodejs/Product/TestAdapter/TestFrameworks/TestFramework.cs

```diff
@@ -111,6 +111,11 @@ class TestFramework {
             return path;
         }
 
+        private string WrapTestNameWithQuot(string path)
```

Comment:
Minor spelling/grammar error - WrapTestNameWithQuotes 

We should also correct the other method name as well while we're at it